Installer/51-gcadapter.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="057e", ATTRS{idProduct}=="0337", MODE="0666"
```

Comment:
Allowing user to set the group via variable, looking for commonly used groups, and allowing the user to explicitly opt-in (again, via a var) to a 0666 rule seems better?

Also, informing the user what's going on during install, and why?

Is there plans to do something like this in the distributed Ubuntu builds in post-install (or similar...)?